.github/workflows/verify-links.yml

```diff
@@ -0,0 +1,69 @@
+name: Verify links
+
+on:
+  push:
+    branches:
+      - main
+      - workflow/verify-links # TODO REMOVE
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "08 08 * * 1"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write # required for peter-evans/create-issue-from-file
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Restore lychee cache
+        uses: actions/cache@v4
+        with:
+          path: .lycheecache
+          key: cache-lychee-${{ github.sha }}
+          restore-keys: cache-lychee-
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v2
+        with:
+          fail: false
```

Comment:
if I understand right this does not affect the following step and its check `if: steps.lychee.outputs.exit_code != 0`